### ISSUE TITLE
underscores

### ISSUE THREAD

**Issue by mle2718** [kimberly-bastille/groundfishRecDST_Shiny#23]
This line
https://github.com/kimberly-bastille/groundfishRecDST_Shiny/blob/64437334701a5c498616240026c410102b577900/app.R#L230
is causing warnings for me. 

```
 Warning message:
Expected 3 pieces. Additional pieces discarded in 5 rows [1, 2, 3, 4, 5]. 
```
This is because I have an underscore in the path
C:/Users/min-yang.lee/Documents/groundfishRecDST_Shiny

I've renamed the folder and I can continue on, but I'm putting this here just in case someone else has a problem one day.

**Comment by mle2718** [comment]
I'm actually getting a weird looking run number now.  
![Screenshot (162)](https://github.com/user-attachments/assets/3be9e2e0-d27d-4cb2-9634-2fc65eb62ddb)

You've probably gotten this also. It's happening because sometimes there are 3 things in the split, and sometimes there are 4.  

A few solutions
1. Use a hyphen to join the run name to the date so, ``output_Test_20241126_145336`` becomes ``output-Test_20241126_145336`.`
2. Put the run name at the end, with or without a hyphen
3. Force every scenario to have a run name (we can just make one up if a user doesn't supply)
4. Save the run name in the data and don't bother trying to parse the filename.

**Comment by kimberly-bastille** [comment]
issue fixed in commit c935456f3066f9f34a100feb6587b53231f08ade

Name converted to using dashes in place of underscores. 

`Run_Name <- function(){`
    `if(stringr::str_detect(input$Run_Name, "_")){`
      `Run_Name <-  gsub("_", "-", input$Run_Name)`
    `}else {`
      `Run_Name <- input$Run_Name`
   ` }`
    `return(Run_Name)`
 ` }`
